xml/ay_bootloader.xml

```diff
@@ -24,9 +24,11 @@
   documentation that comes with your distribution in
   <filename>/usr/share/doc/packages/autoyast2/</filename>
  </para>
-
+<para>
+     By default, &ay; proposes the same booting mechanism as used by the booting medium. For example, if you boot using EFI, the GRUB 2 for EFI is installed. Therefore, you can omit this section, unless you have specific requirements. As the EFI boot requires a specific partitioning, we recommend to use the automatic partitioning as described in <xref linkend="CreateProfile-Automatic-Partitioning"/>, which will create all needed partitions automatically.
```

Comment:
Maybe I would mention there that also bootloader is smart enough that you define only what you want to overwrite and rest is taken from proposal. It use merge strategy, so e.g. if you want just to always have efi and secure boot disabled, it is enough to define just those two settings and rest uses proposed defaults.
But it is just note, so feel free to merge it as it is.